utils/bazel/llvm-project-overlay/mlir/BUILD.bazel

```diff
@@ -8643,7 +8643,11 @@ gentbl_cc_library(
 
 cc_library(
     name = "BytecodeOpInterface",
-    srcs = ["lib/Bytecode/BytecodeOpInterface.cpp"],
+    srcs = [
+        "include/mlir/Bytecode/BytecodeReader.h",
```

Comment:
Checked and we can drop this header from being included in BytecodeOpInterface, the other one we should probably refactor (e.g., extract the config out) so that the cmake and bazel builds are more similar.

Would you mind doing that along with this? (else I can too)

Reply:
I could take a crack at it, but I suspect it would be easier for you to do so given that I'm not terribly familiar with this codebase. I just ran across this with a project that happened to transitively depend on the llvm bazel rules.